crates/core/src/models/event.rs

```diff
@@ -70,7 +70,8 @@ impl ConvertibleStarknetEvent for StarknetEvent {
             })
             .collect::<Result<_, _>>()?;
 
-        let data: Bytes = vec_felt_to_bytes(self.0.data);
+        let data: Bytes =
```

Comment:
I think you can either do `Bytes::from` but then you don't have to type data, or you type data and then you can call `.into()` on you result, no need for both